p2p/ipld/plugin/nodes/nodes.go

```diff
@@ -48,6 +53,11 @@ func init() {
 		nmtHashSize,
 		sumSha256Namespace8Flagged,
 	)
+	// this should already happen when the plugin is injected but it doesn't for some CI tests
+	format.DefaultBlockDecoder.Register(Nmt, NmtNodeParser)
+	// register the codecs in the global maps
+	cid.Codecs[NmtCodecName] = Nmt
+	cid.CodecToStr[Nmt] = NmtCodecName
```

Comment:
Should we panic in case the codec already exists? (e.g. because another project registered `0x7700` for sth else)